File: Session_01_InputAndOutput/java/basics.java
import java.util.Scanner;

public class basics {
    public static void main(String[] args) {
        // 1. Output
        System.out.println("Hello from Java!");

        // 2. Variables
        int age = 19;
        double gpa = 3.75;
        char grade = 'A';
        String name = "Budi";

        System.out.println("Name: " + name);
        System.out.println("Age: " + age + " | GPA: " + gpa + " | Grade: " + grade);

        // 3. Input
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a new age: ");
        age = scanner.nextInt();

        System.out.println("Updated Age: " + age);
        scanner.close();
    }
}

File: Session_01_InputAndOutput/kotlin/basics.kt
import java.util.Scanner

fun main() {
    // 1. Output
    println("Hello from Kotlin!")

    // 2. Variables
    var age = 19
    val gpa = 3.75
    val grade = 'A'
    val name = "Budi"

    println("Name: $name")
    println("Age: $age | GPA: $gpa | Grade: $grade")

    // 3. Input
    val scanner = Scanner(System.`in`)
    print("Enter a new age: ")
    age = scanner.nextInt()

    println("Updated Age: $age")
}
